assets/css/basic_checkout_mercadopago.css

```diff
@@ -278,6 +277,20 @@
 	color: #333333 !important;
 }
 
+.mp-terms-and-conditions {
```

Comment:
Já que todos os atributos dessa classe são !important , talvez faça sentido trocar a hierarquia de **class** (.mp-terms-and-conditions) para **id** (#mp-terms-and-conditions) pra ver se da pra reduzir a quantidade de !important